ush/config.sh.RRFS_dev1

```diff
@@ -31,6 +31,8 @@ PREDEF_GRID_NAME=RRFS_CONUS_3km
 ADDNL_OUTPUT_GRIDS=( "hrrr" )
 
 DO_DACYCLE="true"
+DO_RETRO="false"
+LBCS_ICS_ONLY="false"
```

Comment:
These should be the default, and shouldn't need to be set here, right?

Reply:
Again, I would like to leave them as a comment to make retro run easier. many colleagues may not know those options.